### PR TITLE
[New ruleset] hku.hk

### PR DESCRIPTION
**Pull request by Riotism**

### CODE REVIEW

File: src/chrome/content/rules/hku.hk.xml [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+	<target host="www.visitorcentre.hku.hk" />
+	<target host="webmail.hku.hk" />
+
+	<rule from="^http://(?:www\.)?cedars\.hku\.hk/" to="https://www.cedars.hku.hk/" />
+	<rule from="^http://(?:www\.)?centennialcollege\.hku\.hk/" to="https://www.centennialcollege.hku.hk/" />
+	<rule from="^http://(?:www\.)?fbe\.hku\.hk/" to="https://www.fbe.hku.hk/" />
+	<rule from="^http://(?:www\.)?hkupop\.hku\.hk/" to="https://www.hkupop.hku.hk/" />
+	<rule from="^http://(?:www\.)?its\.hku\.hk/" to="https://www.its.hku.hk/" />
+	<rule from="^http://(?:www\.)?med\.hku\.hk/" to="https://www.med.hku.hk/" />
+	<rule from="^http://(?:www\.)?ppaweb\.hku\.hk/" to="https://www.ppaweb.hku.hk/" />
```
Comment (J0WI) [inline src/chrome/content/rules/hku.hk.xml]: Please avoid non-capturing groups.

**Comment by J0WI** [comment]
I have no idea why this test is failing.
cc @jeremyn @gloomy-ghost @Hainish

**Comment by jeremyn** [comment]
There are duplicate `target`s in the ruleset for `versitech.hku.hk`, [here](https://github.com/Riotism/https-everywhere/blob/963cc0e3ac93279b9a859a209175f01df35829cc/src/chrome/content/rules/hku.hk.xml#L315) and [here](https://github.com/Riotism/https-everywhere/blob/963cc0e3ac93279b9a859a209175f01df35829cc/src/chrome/content/rules/hku.hk.xml#L317).

**Comment by J0WI** [comment]
Oh sure, I looked on the wrong commit.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**